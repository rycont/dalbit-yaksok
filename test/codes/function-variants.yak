약속, (음식)을/를 (사람)와/과 먹기
    "맛있는 " + 음식 + ", " + 사람 + "의 입으로 모두 들어갑니다." 보여주기

"피자"를 "철수"와 먹기
"햄버거"를 "영희"와 먹기
"치킨"을 "형님"과 먹기
"초밥"을 "동생"과 먹기

먹을_음식: "유부초밥"
먹일_사람: "현수"

먹을_음식 을 먹일_사람 과 먹기
먹을_음식 을 (먹일_사람)과 먹기

먹을_음식: "대나무"

(먹을_음식)을 먹일_사람 과 먹기

먹일_사람: "지우"

(먹을_음식)을 (먹일_사람)과 먹기