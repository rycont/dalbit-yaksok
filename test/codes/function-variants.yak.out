맛있는 피자, 철수의 입으로 모두 들어갑니다.
맛있는 햄버거, 영희의 입으로 모두 들어갑니다.
맛있는 치킨, 형님의 입으로 모두 들어갑니다.
맛있는 초밥, 동생의 입으로 모두 들어갑니다.
맛있는 유부초밥, 현수의 입으로 모두 들어갑니다.
맛있는 유부초밥, 현수의 입으로 모두 들어갑니다.
맛있는 대나무, 현수의 입으로 모두 들어갑니다.
맛있는 대나무, 지우의 입으로 모두 들어갑니다.
그건 제가 도와드릴 수 없어요
그건 제가 도와드릴 수 없어요
그건 제가 도와드릴 수 없어요
그건 제가 도와드릴 수 없어요
그것도 제가 도와드릴 순 없어요..
그것도 제가 도와드릴 순 없어요..
그것도 제가 도와드릴 순 없어요..
그것도 제가 도와드릴 순 없어요..
그것도 제가 도와드릴 순 없어요..
그것도 제가 도와드릴 순 없어요..
